relayer/chains/wasm/provider.go

```diff
@@ -181,7 +181,7 @@ func (p *Provider) call(ctx context.Context, message *relayTypes.Message) (*sdkT
 				return nil, fmt.Errorf("failed to handle sequence mismatch error: %v || %v", mmErr, err)
 			}
 		}
-		return nil, err
+		return p.sendMessage(ctx, msgs...)
```

Comment:
Any reason to retry with no delay immediately after error here ?

Reply:
This is a failed transaction and can be immediately fixed with the correct sequence so that we don't start over again.

Reply:
Where is the sequence corrected?  For that case, shouldn't this be inside the if statement above ?

Reply:
yes, corrected above but placed wrong.